tempfork/gliderlabs/ssh/README.md

```diff
@@ -58,7 +58,7 @@ Also, you can [join our Slack](http://slack.gliderlabs.com) to discuss as well.
 
 ## Sponsors
 
-Become a sponsor and get your logo on our README on Github with a link to your site. [[Become a sponsor](https://opencollective.com/ssh#sponsor)]
+Become a sponsor and get your logo on our README on GitHub with a link to your site. [[Become a sponsor](https://opencollective.com/ssh#sponsor)]
```

Comment:
brand -- possibly a vendored item. Happy to drop, or to try to upstream (although I have a queue so there's no guarantee of speed).

Reply:
Let's not touch the tempfork directory